package.json

```diff
@@ -78,7 +77,8 @@
     "source-map-support": "^0.5.19",
     "typescript": "^4.1.4",
     "worker-threads-pool": "^2.0.0",
-    "workerpool": "^6.1.0"
+    "workerpool": "^6.1.0",
+    "sonar-scanner": "^3.1.0"
```

Comment:
You should use `npm install ...` commands to add dependencies, this way the dependencies are always sorted correctly by alphabetical order

Reply:
Done now thankssss